content/blog/D0001-openage_mod_api_game_entity.md

```diff
@@ -0,0 +1,57 @@
+Title: Openage modding API - Units, Buildings & more
+Date: 2018-08-26
+Tags: nyan, modding, API
+Authors: heinezen
+Summary: About the representation of game world objects
+
+Last week we explored the overall principles and design decision for our modding API. This time we will see how the definition of units, buildings and other visible objects in the game world will work.
```

Comment:
you can link to the previous article with `[linktext]({filename}/blog/D0000-articlename.md)`